backend/app.py

```diff
@@ -172,6 +177,47 @@ class CsetsUpdate(BaseModel):
     row_index_data_map: Dict[int, Dict[str, Any]] = {}
 
 
+@app.get("/concept-set-names")
+@app.get("/datasets/csets/names")
+@app.get("/jq-cset-names")
+def cset_names() -> Union[Dict, List]:
+    """Get concept set names"""
+    return csets_read(field_filter=['conceptSetNameOMOP'])
+
+
+@app.get("/datasets/csets")
+def csets_read(field_filter: Union[List[str], None] = Query(default=None)) -> Union[Dict, List]:
```

Comment:
This is the function that does the work of the alias routes:
```py
@app.get("/concept-set-names")
@app.get("/datasets/csets/names")
@app.get("/jq-cset-names")
```

If you want to use this route directly to get concept set names, the following will work: http://localhost:8000/datasets/csets?field_filter=conceptSetNameOMOP

If we want to add multiple filters, you would do it like this:
http://localhost:8000/datasets/csets?field_filter=FIELD1&field_filter=FIELD2

Previously, I've seen (and liked) this approach instead for having a param as a list:
http://localhost:8000/datasets/csets?field_filter=FIELD1,FIELD2
We could do it this way by splitting on `,` to get a list, but it is not the canonical way to do it in FastAPI and doesn't have the native support.

I haven't thought about what this might look like if we wanted do operations on nested fields, but we don't have to supply all of that JQ flexibility / functionality to the user via URL query params. Would be nice, though.